### ISSUE TITLE
Toxic Exposure Summary Screen - adjust screen reader experience for date ranges

### ISSUE THREAD

**Issue by mengA6** [department-of-veterans-affairs/va.gov-team#81395]
## Issue Description

discovered in [Toxic Exposure: Gulf War 1990 Summary page#67018](https://app.zenhub.com/workspaces/disability-experience-63dbdb0a401c4400119d3a44/issues/gh/department-of-veterans-affairs/va.gov-team/67018)

On summary screen states like

<img width="238" src="https://api.zenhub.com/attachedFiles/eyJfcmFpbHMiOnsibWVzc2FnZSI6IkJBaHBBN1VLQkE9PSIsImV4cCI6bnVsbCwicHVyIjoiYmxvYl9pZCJ9fQ==--0153f4d90400998957b6f5b6b51928706b11b981/image.png" alt="image.png" />

and

<img width="261" src="https://api.zenhub.com/attachedFiles/eyJfcmFpbHMiOnsibWVzc2FnZSI6IkJBaHBBNjRLQkE9PSIsImV4cCI6bnVsbCwicHVyIjoiYmxvYl9pZCJ9fQ==--0b3b427d031fca67806969755b3400afc1c47f25/image.png" alt="image.png" />

Adjust the screen reader experience to make it clear what represents a start date (or the lack of a start date entered) and what represents an end date (or the lack of an end date entered).

Current screen reader experience as demoed by Christine (not sure which screen reader - VoiceOver?) just speaks "no start date entered (pause) 2003" for the first example. 

[Figma screens](https://www.figma.com/file/dBKlB23Hs2oa53euXcXmmK/526-toxic-exposure?type=design&node-id=1236-55054&mode=design&t=m6ByF6kbzyFMozzE-11)

---
## Tasks
- [ ] Consult with CAIA and/or other teams that have implemented a precedent to find guidance on the screen reader experience
- [ ] Update design (with annotations)
- [ ] Check approach with engineers
- [ ] Document changes in an implementation ticket and post in slack

## Acceptance Criteria
- [ ] Above tasks completed
---
## How to configure this issue
- [ ] **Attached to a Milestone** (when will this be completed?)
- [ ] **Attached to an Epic** (what body of work is this a part of?)
- [ ] **Labeled with Team** (`product support`, `analytics-insights`, `operations`, `service-design`, `Console-Services`, `tools-fe`)
- [ ] **Labeled with Practice Area** (`backend`, `frontend`, `devops`, `design`, `research`, `product`, `ia`, `qa`, `analytics`, `contact center`, `research`, `accessibility`, `content`)
- [ ] **Labeled with Type** (`bug`, `request`, `discovery`, `documentation`, etc.)

**Comment by pacerwow** [comment]
Would we pass staging review if we didn't do this?
